treescript/docker.d/worker.yml

```diff
@@ -8,7 +8,7 @@ hg_share_base_dir: { "$eval": "HG_SHARE_BASE_DIR" }
 upstream_repo: { "$eval": "UPSTREAM_REPO" }
 merge_day_clobber_file: { "$eval": "MERGE_DAY_CLOBBER_FILE" }
 hg_ssh_config:
-  $if: 'COT_PRODUCT == "firefox"'
+  $if: 'COT_PRODUCT in ["firefox", "thunderbird"]'
```

Comment:
This condition was added https://github.com/mozilla-releng/scriptworker-scripts/pull/289/files#diff-add463d8fe89da73274d3b5dad502ae900285533e91c4003506f25c15d313910R11. It was obviously too restrictive.